config/routes.rb

```diff
@@ -1,13 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # scope 'users/:user_id', as: 'user' do
-  #   resources :posts, only: [:index, :show]
-  # end
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create]
+      resources :likes, only: [:create, :destroy]
+    end
   end
-  # resources :users, only: [:index, :show]
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
Hi @FrederickMih,
Thanks for the review and especially for the hints :star_struck: 
This is one of the best reviews I have received since the beginning of the program :smile: 
For a couple of months now, I fill like as far as everything is working, there just approved the project without checking the code :smiling_face_with_tear: